acceptance/Rakefile

```diff
@@ -7,7 +7,7 @@ require 'beaker-hostgenerator'
 
 $LOAD_PATH << File.expand_path(File.join(File.dirname(__FILE__), 'lib'))
 require 'puppet/acceptance/git_utils'
-extend Puppet::Acceptance::GitUtils
+extend ::Puppet::Acceptance::GitUtils
```

Comment:
Having to prefix an extend namespace with `::` is a really uncommon idiom.  This seems like a code smell to me, but digging in a but further to understand what's being described here.